OPERATOR.rst

```diff
@@ -173,28 +173,61 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
-
-    _select dev.gitlab
-    cd terraform/gitlab
-    make apply
-
-It may be necessary to set ``CI_COMMIT_REF_NAME=develop`` to work around
-``check_branch``.
-
-The GitLab instance should be online again in 10 minutes or so. If it takes
-substantially longer, contact the lead.
+Operators must check for updates to GitLab on a weekly basis. Compare the
+current version of GitLab found at the ``/help`` endpoint for `GitLab dev`_ to
+the available releases for the `GitLab Docker image`_. When updating the GitLab
+instance, check if there are applicable updates to the `GitLab runner image`_.
+Use the latest runner image whose major and minor version match that of the
+GitLab image.
+
+Before starting the update process, check the `GitLab release notes`_ for
+upgrading instructions. When upgrading across multiple GitLab versions, follow
+the prescribed GitLab `upgrade path`_.
+
+.. _GitLab dev: https://gitlab.dev.singlecell.gi.ucsc.edu/help
+.. _GitLab Docker image: https://hub.docker.com/r/gitlab/gitlab-ce/tags
+.. _GitLab runner image: https://hub.docker.com/r/gitlab/gitlab-runner/tags
+.. _GitLab release notes: https://about.gitlab.com/releases/categories/releases/
+.. _upgrade path: https://docs.gitlab.com/ee/update/index.html#upgrade-paths
+
+Before any changes are applied, run::
+
+	git fetch -all
+	git checkout -b gitlab/yyyy-mm-dd github/develop
```

Comment:
The branch name contradicts what's in the the PR checklist.

Please add a checklist for this type of PR.